packages/web-components/src/text-input/README.md

```diff
@@ -0,0 +1,134 @@
+# TextInput
+
+> An implementation of a [text input](https://w3c.github.io/html-reference/input.text.html) as a form-connected web-component.
+
+<br />
+
+## **Design Spec**
+
+[Link to Text Input Design Spec in Figma](https://www.figma.com/file/TvHmWjZYxwtI9Tz6v5BT7E/Input?node-id=2366-657&t=UNSOfCD3St9ffppx-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 Text Input extends from the [FAST Text Field](https://explore.fast.design/components/fast-text-field) and is intended to be as close to the Fluent UI React 9 Input implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+<br />
+
+## Class: `TextInput`
+
+<br />
+
+### **Component Name**
+
+`<fluent-text-input>`
+
+<br />
+
+### **Variables**
+
+| Name                  | Description                          | Type                                                                                                             |
+| --------------------- | ------------------------------------ | ---------------------------------------------------------------------------------------------------------------- |
+| `TextInputSize`       | Size variations for text input       | `{ small: "small", medium: "medium", large: "large" }`                                                           |
+| `TextInputAppearance` | Appearance variations for text input | `{ outline: "outline", underline: "underline", filledLighter: "filled-lighter", filledDarker: "filled-darker" }` |
+| `TextInputType`       | Text input types                     | `{ email: "email", password: "password", tel: "tel", text: "text", url: "url" }`                                 |
```

Comment:
There are more types in FUIR9:
https://github.com/microsoft/fluentui/blob/3c54de58895744516ed7befb629d77bac039a9e2/packages/react-components/react-input/src/components/Input/Input.types.ts#L93-L105

Reply:
FUIR9 docs:
> Note that no custom styling is currently applied for alternative types, and some types may activate browser-default styling which does not match the Fluent design language.

I do not have a preference here, we should just document the diferrences.

Reply:
Was counting on a preference! 😁

Reply:
Documented the deltas